monitoring.adoc

```diff
@@ -777,17 +777,83 @@ account activity for one user without filling up mailbox.log with log
+
+image:images/doublesemicolon.png[ua value with double semi-colon]
+
+The following key value pairs are currently supported and may be recorded in log entries in any order and eny combination:
+
+* `ip` - IP of the TCP/IP client making a request
+* `oip` - originating IP address. When a request is made through NGINX proxy, this value will contain the IP address of the client application, while `ip` value will contain the IP address of the proxy server
+* `cid` - connection id of a server that is monotonically increasing - useful for tracking individual connections
+* `id` - ID of the target account
+* `name` - name of the target account (email address)
+* `aid` - ID of the authenticated account. Only present if target account is different then authenticated account
+* `aname` - name of the authenticated account. Only present if target account is different then authenticated account
+* `mid` - Id of requested mailbox. Only present if request is dealing with a mailbox
+* `ua`  - name of the client application (i.e. User Agent)
+* `via` - list of IP addresses and user-agents of the request's proxy chain
+* `soapId` - ID assigned to a SOAP request to track proxied hops for a particular request
+* `msgid` - value of Message-ID header of the message being operated on
+* `ds` - name of the Data Source being operated on
+* `port` - server port to which the client connected
+* `oport` - originating port number of request
+* `oproto` - originating protocol of request. This can be passed by internal components that make SOAP requests on behalf of a user (e.g. MTA and IMAPd)
+
+The example below is a record showing that
+on October 25, 2017, 28 minutes after midnight, a POP3 client with IP address `222.173.186.17` has contacted the {product-name} server and that the request was proxied through a local proxy server with IP 10.1.1.136.
+
+image:images/poplogmessage.png[Mailbox Log Entry for POP]
+
+The following example shows a record of a failed `IMAP STATUS` request sent by `user1@mydomain.com` using AquaMail mobile app. The user's device has IP address `72.83.144.255` (as reported in `oip` field). The request came to IMAP server via Zimbra nginx proxy, which has IP address `10.4.4.138` (as reported in `ip` and `via` fields).
+
+image:images/imaplogmessage.png[Mailbox Log Entry for IMAP]
+
+The following example shows a record of LMTP server delivering a message. The IP address in this log message most likely belongs to Zimbra MTA running on local network.
```

Comment:
Change "Zimbra MTA running on local" to "{product-name} MTA running on the local"?